test/node/resendable-call-from-client.js

```diff
@@ -11,10 +11,16 @@ const interfaces = {
   iface: {
     method: (connection, callback) => {
       callsReceivedByServer++;
-      if (callsReceivedByServer === 2) {
-        callback(null);
-      } else if (callsReceivedByServer > 2) {
-        test.fail('Server must not receive more than two call messages');
+      switch (callsReceivedByServer) {
```

Comment:
I think if/else would look better here, though as you want.
Also :+1: for extensive checks.